### PR TITLE
Use end-coords-interpolation in picking

### PR DESCRIPTION
**Pull request by pazeshun**
Updated version of #2425

### CODE REVIEW

**Comment by pazeshun** [comment]
Tested in real robot.
Currently there is no problem, but we have to revert this when strange motion happens.

File: jsk_arc2017_baxter/euslisp/lib/arc-interface.l [hunk 1]
```diff
@@ -602,27 +603,17 @@
...
-                    (pushback (send *baxter* arm :inverse-kinematics ec) avs)
-                    (send ec :translate #f(0 0 -30) :world)
+                    (send ec :translate #f(0 0 -330) :world)
                     (pushback (send *baxter* arm :inverse-kinematics ec) avs)))
                 (setq avs (remove nil avs))
-                (if avs
-                  (progn
+                (when avs
+                  (send *ri* :update-robot-state :wait-until-update t)
+                  (when (null (send *ri* :angle-vector-sequence-raw avs
```
Comment (wkentaro) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: unless

File: jsk_arc2017_baxter/euslisp/lib/arc-interface.l [hunk 2]
```diff
@@ -861,7 +852,11 @@
       (unless av
         (ros::ros-error "[:try-to-pinch-object] arm:~a IK to object fails. Abort picking" arm)
         (return-from :try-to-pinch-object :ik-fail))
-      (send *ri* :angle-vector-raw av 3000 (send *ri* :get-arm-controller arm) 0)
+      (send *ri* :update-robot-state :wait-until-update t)
+      (if (null (send *ri* :angle-vector-raw av
```
Comment (wkentaro) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: unless

**Comment by wkentaro** [comment]
```

+ rosdep install -q -y --rosdistro indigo -n -q --ignore-src --skip-keys=jsk_smart_gui --skip-keys=ros3djs --skip-keys=pr2_calibration_launch --skip-keys=jsk_android_gui_api9 --skip-keys=ros2djs --skip-keys=face_recognition --skip-keys=roslibjs --skip-keys=force_proximity_ros --skip-keys=safe_teleop_base --from-paths /opt/ros/indigo/share .
dpkg-query: no packages found matching ros-indigo-convex-decomposition
dpkg-query: no packages found matching ros-indigo-pr2-moveit-plugins
dpkg-query: no packages found matching unzip
dpkg-query: no packages found matching ros-indigo-baxtereus
dpkg-query: no packages found matching libgstreamer-plugins-base0.10-dev
dpkg-query: no packages found matching libgstreamer0.10-dev
dpkg-query: no packages found matching ros-indigo-openhrp3
dpkg-query: no packages found matching ros-indigo-moveit-resources
dpkg-query: no packages found matching ros-indigo-jsk-pcl-ros-utils
dpkg-query: no packages found matching python-mock
dpkg-query: no packages found matching xvfb
dpkg-query: no packages found matching ros-indigo-jsk-pcl-ros
dpkg-query: no packages found matching ros-indigo-jsk-baxter-startup
dpkg-query: no packages found matching python-pygraphviz
dpkg-query: no packages found matching libnetpbm10-dev
dpkg-query: no packages found matching ros-indigo-pr2-moveit-config
dpkg-query: no packages found matching ros-indigo-ivcon
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
safe_teleop_base: Cannot locate rosdep definition for [pcl]
+ EXIT_STATUS=1
+ find -L . -name manifest.xml.deprecated
```

**Pull request merged**